cob_frame_tracker/src/cob_frame_tracker.cpp

```diff
@@ -210,6 +211,10 @@ void CobFrameTracker::publish_twist(ros::Duration period)
 {
 	tf::StampedTransform transform_tf;
 	geometry_msgs::TwistStamped twist_msg;
+
+    std_msgs::Float64MultiArray error_msg;
```

Comment:
The problem is:
- Before: Tab-Indentation
- Now: 4-Space-Indentation

If desired then replace everything with 4 space indentation.